02/PrintRecords.cpp

```diff
@@ -7,95 +7,141 @@ using namespace std;
 
 namespace samples
 {
-	void InputRecords(Record* info)
+	Record InputRecord(Record record)
 	{
 		cout << "First name: ";
-		cin >> info->FirstName;
+		cin >> record.FirstName;
 		
 		cout << "Last name: ";
-		cin >> info->LastName;
+		cin >> record.LastName;
 		
 		cout << "Student ID: ";
-		cin >> info->StudentID;
+		cin >> record.StudentID;
 
 		cout << "Score:  ";
-		cin >> info->Score;
+		cin >> record.Score;
+
+		return record;
 	}
 
 	void ManageRecordsExample()
 	{
-		Record studentInfos[5];
-		ofstream fileStream("studentRecords.dat", ios_base::out);
+		Record record;
+
+		ofstream outputStream("studentRecords.dat");
+		ifstream inputStream("studentRecords.dat");
 
-		int existingStudents = 1;
+		int recordCount = 0;
 
-		studentInfos[0].FirstName = "Pope";
-		studentInfos[0].LastName = "Kim";
-		studentInfos[0].StudentID = "A12345678";
-		studentInfos[0].Score = 80;
+		record.FirstName = "Pope";
+		record.LastName = "Kim";
+		record.StudentID = "A12345678";
+		record.Score = "80";
 
-		PrintRecords(&studentInfos[0], &fileStream, existingStudents);
+		WriteFileRecord(&outputStream, record, true, &recordCount);
```

Comment:
&recordCount? ... why?
++recordCount; is fine

&outputStream.... why? use reference